Formula/f/freeswitch.rb

```diff
@@ -114,8 +114,11 @@ class Freeswitch < Formula
 
   resource "libks" do
     url "https://github.com/signalwire/libks.git",
-        tag:      "v2.0.2",
-        revision: "423c68c92d1871e2c9cb83d974eac7830ddcdcf5"
+        tag:      "v2.0.6",
+        revision: "3bc8dd0524a865becdd98c3806735eb306fe0a73"
+
+    # Fix compile with newer Clang, https://www.redhat.com/en/blog/new-warnings-and-errors-clang-15
+    patch :DATA if DevelopmentTools.clang_build_version >= 1500
```

Comment:
Thanks, can you send this patch upstream? (Or have you done that already?)

Reply:
- https://github.com/signalwire/libks/issues/217

Noted. :)